src/react-app/src/configs/i18n.ts

```diff
@@ -0,0 +1,23 @@
+import i18n from 'i18next';
+import Backend from 'i18next-http-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import { initReactI18next } from 'react-i18next';
+
+const backendOptions = {
+  loadPath: '/i18n/{{lng}}.json'
```

Comment:
Using an absolute loadPath here won't work because of how we upload & deploy our webapps. With absolute paths, it tries to load either from the CDN root (see 1st image) or the index file root (which would be https://apps.{domain}/i18n/...` – see second image). 

![image](https://user-images.githubusercontent.com/32391370/120002144-82808500-bfa2-11eb-90f5-fe0c5b7adbd8.png)

![image](https://user-images.githubusercontent.com/32391370/120004150-73023b80-bfa4-11eb-974c-ddb15c66a239.png)

Not sure if React can/will replace the `PUBLIC_URL` in `tsx` files, but the CDN_URL is being passed into the build [in the Jenkinsfile](https://github.com/MyPureCloud/genesys-cloud-client-auth/blob/main/Jenkinsfile#L36). The CDN_URL always ends with a backslash so you should be able to do something like: 

``` ts
loadPath: `{PUBLIC_URL}i18n/{{lng}}.json`
```

You may need to add some more options to the backendOptions, but you would have to test it out. By loading the translation file from a full URL, it will ensure it can always load regardless of where it is deployed to: 
* `apps.{host}.com/client-auth/` (app root)
* `apps.{host}.com/client-auth/{version}/` (app version root)
* `{cdn_url}/client-auth/{version}-{buildNum}/` (cdn deploy url)

**Testing it locally:** Since you already have some translation files in the CDN now, you could use this locally to make sure it can make the GET request: 

``` ts
loadPath: `https://dhqbrvplips7x.cloudfront.net/client-auth/PCM-1645---Translations/Config-2/i18n/{{lng}}.json`
```